catalog/security/catalog-security-pointofcontact-policyplugin/src/test/java/org/codice/ddf/catalog/security/policy/metacard/PointOfContactPolicyPluginTest.java

```diff
@@ -0,0 +1,212 @@
+        when(mockOperationTransaction.getPreviousStateMetacards()).thenReturn(Collections.emptyList());
+
+        Map<String, Serializable> inputProperties = setupAndGetInputProperties();
+        Metacard metacard = new MetacardImpl();
+
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreUpdate(metacard,
+                inputProperties);
+
+        responseIsEmpty(response);
+    }
+
+    @Test
+    public void processPreUpdateDoesNothingWhenPointOfContactsAreSame() throws java.lang.Exception {
+        List<Metacard> listWithMetacard = new ArrayList<Metacard>();
+        MetacardImpl metacard = new MetacardImpl();
+        metacard.setId(TEST_ID);
+        metacard.setPointOfContact(TEST_POINT_OF_CONTACT);
+        listWithMetacard.add(metacard);
+
+        when(mockOperationTransaction.getPreviousStateMetacards()).thenReturn(listWithMetacard);
+
+        Map<String, Serializable> inputProperties = setupAndGetInputProperties();
+
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreUpdate(metacard,
+                inputProperties);
+
+        responseIsEmpty(response);
+    }
+
+    @Test
+    public void processPreUpdateReturnsPolicyWhenPointOfContactsAreDifferent()
+            throws java.lang.Exception {
+        List<Metacard> listWithMetacard = new ArrayList<Metacard>();
+        MetacardImpl oldMetacard = new MetacardImpl();
+        oldMetacard.setId(TEST_ID);
+        oldMetacard.setPointOfContact("edited-" + TEST_POINT_OF_CONTACT);
+        listWithMetacard.add(oldMetacard);
+
+        when(mockOperationTransaction.getPreviousStateMetacards()).thenReturn(listWithMetacard);
+
+        Map<String, Serializable> inputProperties = setupAndGetInputProperties();
+
+        MetacardImpl newMetacard = new MetacardImpl();
+        newMetacard.setId(TEST_ID);
+        newMetacard.setPointOfContact(TEST_POINT_OF_CONTACT);
+
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreUpdate(newMetacard,
+                inputProperties);
+
+        responseHasPolicy(response);
+    }
+
+    @Test
+    public void processPreUpdateReturnsPolicyWhenOnePointOfContactIsNull()
+            throws java.lang.Exception {
+        List<Metacard> listWithMetacard = new ArrayList<Metacard>();
+        MetacardImpl oldMetacard = new MetacardImpl();
+        oldMetacard.setId(TEST_ID);
+        listWithMetacard.add(oldMetacard);
+
+        when(mockOperationTransaction.getPreviousStateMetacards()).thenReturn(listWithMetacard);
+
+        Map<String, Serializable> inputProperties = setupAndGetInputProperties();
+
+        MetacardImpl newMetacard = new MetacardImpl();
+        newMetacard.setId(TEST_ID);
+        newMetacard.setPointOfContact(TEST_POINT_OF_CONTACT);
+
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreUpdate(newMetacard,
+                inputProperties);
+
+        responseHasPolicy(response);
+    }
+
```

Comment:
❓ Should we test the reverse case, i.e., old metacard has a POC but the new one doesn't?